source/supported-operations.txt

```diff
@@ -43,6 +43,19 @@ Comparison Functions and Operators
      - Return the largest argument
    * - ``IN()``
      - Check whether a value is within a set of values
+   * - ``INTERVAL(N,N1,N2,...)``
+     - Return 0-based index position of last number in the list to be ``< N``.
+       Return ``-1`` if ``N`` is ``NULL``
+     
+       ``N1``, ``N2``, etc. must be given in ascending order. 
```

Comment:
What happens when they are not specified in ascending order? I assume an error is thrown? Do we want to specify this?

Reply:
No idea. This is pulled from [MySQL](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#function_interval).